app/controllers/repositories_controller.rb

```diff
@@ -109,8 +109,12 @@ def destroy_info
   end
 
   def destroy
-    @project.repository.destroy
-    flash[:notice] = I18n.t('repositories.delete_sucessful')
+    repository = @project.repository
+    if repository.destroy
+      flash[:notice] = I18n.t('repositories.delete_sucessful')
+    else
+      flash[:error] = repository.errors.full_messages.join("\n")
```

Comment:
Strange, as this is used quite a number of controllers (search for `errors.full_messages.join("\n")`). 

What does work is `errors.full_messages`,  as the flash render helper joins them into `<br/>`. (cf. `join_flash_messages`)